### PR TITLE
Update Set-WinSystemLocale.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Added line breaks after headings. 
Added extra line break as it seemed to be the authors intent, but did not render in the web page. 
Added PowerShell connotation to the code blocks.
Removed command prompt from the example.
Corrected links.

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012-ps/internationalcmdlets/Set-WinSystemLocale.md [hunk 1]
```diff
@@ -12,31 +12,36 @@ author: andreabarr
...
 Note that because this is a system setting, this can only be changed by a user with Administrator permissions, and changes do not take effect until the computer is restarted.
 
 ## EXAMPLES
 
 ### Example 1: Set the system locale
-```
-PS C:\>Set-WinSystemLocale -SystemLocale ja-JP
+
+```powershell
+Set-WinSystemLocale -SystemLocale ja-JP
```
Comment (tseward) [inline docset/winserver2012-ps/internationalcmdlets/Set-WinSystemLocale.md:36]: The standard is still to include `PS C:\> ` as part of the cmdlet example.

File: docset/winserver2012-ps/internationalcmdlets/Set-WinSystemLocale.md [hunk 2]
```diff
@@ -52,13 +57,15 @@ Accept wildcard characters: False
...
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### CultureInfo
+
 An object that contains a BCP-47 tag to specify the system locale for the current computer.
-For more information about the CultureInfo object, see CultureInfo Classhttp://go.microsoft.com/fwlink/?LinkID=242306.
+For more information about the CultureInfo object, see [CultureInfo Class](http://go.microsoft.com/fwlink/?LinkID=242306).
```
Comment (tseward) [inline docset/winserver2012-ps/internationalcmdlets/Set-WinSystemLocale.md:68]: Links need to be updated with `https://`

File: docset/winserver2012-ps/internationalcmdlets/Set-WinSystemLocale.md [hunk 3]
```diff
@@ -52,13 +57,15 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (tseward) [inline docset/winserver2012-ps/internationalcmdlets/Set-WinSystemLocale.md:61]: Update this link to `https://`

**Comment by get-itips** [comment]
I am ok once suggestions by other reviewers are applied

**Comment by baardhermansen** [comment]
> I am ok once suggestions by other reviewers are applied

Been away during the weekend, starting to work on a new commit now.

**Pull request merged**